scripts/qemu/auto_qemu_with_usb_storage

```diff
@@ -7,9 +7,11 @@ then
 	echo "$image_file_name found."
 else
 	echo "$image_file_name not found. A new one will be created now..."
+	mkfs -t vfat $image_file_name
```

Comment:
Should this line go after `qemu-img create ...`? If file doesn't exist, mkfs will fail

Reply:
Yes, thank you, fixed.